log/log.go

```diff
@@ -48,7 +48,7 @@ func NewLogger(config config.Log) (*zap.Logger, error) {
 	return logger, nil
 }
 
-func GRPCRequestToLogFields(info *grpc.StreamServerInfo, start time.Time, err error) []zapcore.Field {
+func GRPCRequestToLogFields(_ *grpc.StreamServerInfo, start time.Time, err error) []zapcore.Field {
```

Comment:
__[golint]__
* [golint] exported function GRPCRequestToLogFields should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->